### PR TITLE
Added PL translation for Shortcut settings pane

### PR DESCRIPTION
**Pull request by MarcinOrlowski**

### CODE REVIEW

File: src/main/resources/resources/logisim/strings/gui/gui.properties [hunk 1]
```diff
@@ -609,60 +609,60 @@ windowToolbarHidden = Hidden
 windowToolbarImportant = Important: changing the below values may have unpredictable results!
 windowToolbarLocation = Toolbar location:
 windowToolbarLookandfeel = Look and Feel:
```
Comment (davidhutchens) [inline src/main/resources/resources/logisim/strings/gui/gui.properties:611]: Might as well make Feel lower case too.

**Review by davidhutchens — APPROVED** [review]
Looks good to me. Of course, I didn't actually check the Polish.

I have one minor suggestion.

File: src/main/resources/resources/logisim/strings/gui/gui.properties (hunk 1 above)
Comment (MarcinOrlowski) [reply src/main/resources/resources/logisim/strings/gui/gui.properties:611]: Done

**Review by davidhutchens — DISMISSED** [review]
LGTM

File: src/main/resources/resources/logisim/strings/gui/gui.properties [hunk 2]
```diff
@@ -608,61 +608,61 @@ windowTitle = Window
...
-hotkeyErrMeta = Must contain the %s key.
-hotkeyErrConflict = This key conflicts with: %s.
-hotkeyDirNorth = North
-hotkeyDirSouth = South
-hotkeyDirEast = East
-hotkeyDirWest = West
+hotkeySimReset = Reset simulation
+hotkeySimStep = Single-step propagation
+hotkeySimTickHalf = Manual tick half cycle
+hotkeySimTickFull = Manual tick full cycle
```
Comment (maehne) [inline src/main/resources/resources/logisim/strings/gui/gui.properties:637]: Shouldn't it be "Manually tick..."?

@davidhutchens: Could you maybe comment on this?

**Review by maehne — APPROVED** [review]
Thanks @MarcinOrlowski for this localization work!

File: src/main/resources/resources/logisim/strings/gui/gui.properties (hunk 2 above)
Comment (davidhutchens) [reply src/main/resources/resources/logisim/strings/gui/gui.properties:637]: Now that you point it out, it does seem slightly off. I would actually delete "Manual" and just write "Tick half cycle" and "Tick full cycle". Manual(ly) provides no additional information.

But note that "Manual Tick Half Cycle" is the name of the menu item. So either way, we should be consistent. I might just leave it be.

File: src/main/resources/resources/logisim/strings/gui/gui.properties [hunk 3]
```diff
@@ -608,61 +608,61 @@ windowTitle = Window
...
+hotkeyWindowClose = Close the window
+hotkeyWindowMinimize = Minimize the window
+hotkeyFileExport = Export file
+hotkeyFilePrint = Print file
+hotkeyErrMeta = Must contain the %s key
+hotkeyErrConflict = This key conflicts with: %s
+hotkeyDirNorth = ↑ North
+hotkeyDirSouth = ↓ South
+hotkeyDirEast = → East
+hotkeyDirWest = ← West
```
Comment (juli-99) [inline src/main/resources/resources/logisim/strings/gui/gui.properties:650]: I'm not sure if adding the arrows is beneficial.
```
        ↑ North
← West  ↓ South  → East
```
is IMO not better than
```
      North
West  South  East
```
If we decide that we like the arrows, we should consider moving `→` to the right of `East` (`East →`)

**Review by maehne — DISMISSED** [review]
This looks good to me. The two remaining open remarks are valid, but still merging this PR would be an improvement to the localization.

**Review by davidhutchens — APPROVED** [review]
This still looks good to me.

**Review by maehne — APPROVED** [review]
Looks good to me! There's one unresolved comment, you might consider regading dropping "Manual" in "Manual tick half/full cycle".

**Pull request merged**